src/lib/components/Cytoscape.react.js

```diff
@@ -676,7 +787,50 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
-
+    /**
+     * Define a custom context menu
+     */
+    contextMenu: PropTypes.arrayOf(
+        PropTypes.shape({
+            /**ID of the menu item in the context menu */
+            id: PropTypes.string.isRequired,
+            /**The label on the context menu item*/
+            content: PropTypes.string.isRequired,
+            /**The tooltip text when hoevring on top of a context menu item */
+            tooltipText: PropTypes.string,
+            /***/
+            coreAsWell: PropTypes.string,
+            /** One of 'node' or 'edge', both or neither. This will determine where the context
+             *  menu item will show up.
+             */
+            selector: PropTypes.string,
+            /**Sepcify which js function to use as behaviour for the context menu item
+             * One of 'remove', 'add_node', or 'add_edge'
+             */
+            onClick: PropTypes.string,
+            /**Use a custom js function in a namespace to define context menu item behaviour*/
+            onClickCustom: PropTypes.string,
+        })
+    ),
+    /**
+     * Retrieve relevant data when context menu item is clicked
+     */
+    contextMenuData: PropTypes.exact({
+        /**ID of the menu item in the context menu */
+        menuItemId: PropTypes.string,
+        /**x-position of the context click */
+        x: PropTypes.number,
```

Comment:
Are these `x` and `y` coordinates in pixels, or do they use the same coordinate system as the node positions? (I think it's the latter but would be good to clarify and state here.)